### ISSUE TITLE
Create Remix jsconfig.json still contains references to include remix.env.d.ts, ts, and tsx

### ISSUE THREAD

**Issue by 39digits** [remix-run/remix#2967]
### What version of Remix are you using?

1.4.1

### Steps to Reproduce

1. Run `npx create-remix@latest`
2. Name the project anything you wish.
3. Select "Just the basics"
4. Use "Remix App Server"
5. Yes for npm install
6. Select "JavaScript" as your preferred language
7. Open the generated `jsconfig.json` 

### Expected Behavior

Contents of the `jsconfig.json` file should not contain references to `remix.env.d.ts`, `*.ts`, and `*.tsx` in the include since it is a JavaScript project.

### Actual Behavior

The generated `jsconfig.json` file contains the following include line:

```
"include": ["remix.env.d.ts", "**/*.ts", "**/*.tsx"],
```

This shows as an error in Visual Studio Code

```
No inputs were found in config file '/home/username/remix-project/jsconfig.json'. Specified 'include' paths were '["remix.env.d.ts","**/*.ts","**/*.tsx"]' and 'exclude' paths were '[]'.
```

The `jsconfig.json` file has shown issues from 1.3.0 onwards but is still present in the latest version (1.4.1)

**Comment by 39digits** [comment]
Potentially fixed in #3012

**Comment by MichaelDeBoey** [comment]
Closed by #3529
